docs/tutorials/Fine-Tuning-Semantic-Parsing.md

```diff
@@ -350,3 +347,81 @@ In this post, we showcase the process of fine-tuning a Llama2 model using Levant
 
 Semantic parsing is a critical step in translating user queries into machine-readable, structural format, which is essential for programmatic interactions with AI systems.
 We chose the GEM ViGGO dataset to demonstrate this task. The out-of-box Llama2-7B Chat model struggles to follow the instruction and hallucinates at predicting attributes. By applying fine-tuning, both full-weight and LoRA, we significantly improved the model's ability to perform on this task.
+
+## Appendices
+
+### Appendix A: The Prompt Used in This Task
+
+```
+You are a helpful chatbot to assist users to parse queries in natural language into structural format.
```

Comment:
oh man this is long. I think it's not necessary to use such a long prompt for finetuning (especially I don't think you need few shot examples with fine tuning?)